core/templates/cowdata.h

```diff
@@ -416,6 +407,21 @@ Error CowData<T>::resize(Size p_size) {
 	return OK;
 }
 
+template <typename T>
```

Comment:
Should this be a private method?

Reply:
Yes, i think it already is? It's declared in the private part of the class.

Reply:
No it is [not](https://github.com/godotengine/godot/blob/15288154bb57757f70858c9284fee574b134297f/core/templates/cowdata.h#L169) but neither is a whole pile of other stuff that looks like it should be. Not a bug of this pr.

Reply:
_realloc is declared two lines above the one you linked, in the private block :)

Reply:
Ohhhh. Right, I'm being a big dumbdumb. Thanks